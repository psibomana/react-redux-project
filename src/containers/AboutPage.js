import React from 'react';
import Header from '../components/common/Header';

const AboutPage = () => {
  return (

    <div className="container">
      <Header/>
      <div class="card align v-offset-10">
        <div class="card-body">
          <h1>About</h1>
          <p>This application uses React, Redux, React Router and a variety of other helpful libraries.</p>
        </div>
      </div>
    </div>
  );
}

export default AboutPage;
