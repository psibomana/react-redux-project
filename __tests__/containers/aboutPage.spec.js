import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import AboutPage from '../../src/containers/aboutPage';

describe('The About Page', () => {
  it('should render as expected', () => {
    const aboutPage = shallow(<AboutPage />);

    expect(toJson(aboutPage)).toMatchSnapshot();
  })
});
