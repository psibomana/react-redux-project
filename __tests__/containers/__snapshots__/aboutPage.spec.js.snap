// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The About Page should render as expected 1`] = `
<div
  className="container"
>
  <Header />
  <div
    className="card align v-offset-10"
  >
    <div
      className="card-body"
    >
      <h1>
        About
      </h1>
      <p>
        This application uses React, Redux, React Router and a variety of other helpful libraries.
      </p>
    </div>
  </div>
</div>
`;
